### ISSUE TITLE
Adjust the reflectance of the cloud to get the desired amount of temperature change

### ISSUE THREAD

**Issue by jbphet** [phetsims/greenhouse-effect#82]
As of this writing, the temperature drops about 4 degrees Celsius when the cloud is turned on.  The design team has decided that this is too much, and the code should be adjusted to cause less cooling but to look visually the same on the Waves screen.

**Comment by jbphet** [comment]
I've adjust the model behavior and "Hollywooded" the view to make it still look good even when reflecting less light.

I looked at precise values on this, and the temperature change was actually a little more than 1 degree, so we should review later as a team and decide if it works for us.  In the default position of the slider, the temperature goes down 1 degree Kelvin and 1 degree Celsius, so it seemed pretty decent to me.

**Comment by arouinfar** [comment]
@jessegreenberg this should be included in the prototype version. I'm not sure if these commits were already cherry-picked into the release branch.

**Comment by jessegreenberg** [comment]
I confirmed that changes for this issue were made in the 1.0 branch. However, I noticed that https://github.com/phetsims/greenhouse-effect/commit/55963505b6a09fa0809ce1078495402bd8339d99 was made to 1.0 but **not** to master. @jbphet should https://github.com/phetsims/greenhouse-effect/commit/55963505b6a09fa0809ce1078495402bd8339d99 be in master as well?

**Comment by arouinfar** [comment]
@jessegreenberg this behavior should also be in master.

**Comment by jessegreenberg** [comment]
Thanks @arouinfar. The reason I am unsure about https://github.com/phetsims/greenhouse-effect/issues/82#issuecomment-924440024 is that some of the commits above are in master but https://github.com/phetsims/greenhouse-effect/commit/55963505b6a09fa0809ce1078495402bd8339d99 is not.

However, this is the last issue to verify for https://github.com/phetsims/qa/issues/710, and it is more design related than QA. Can you please verify that it is working correctly in 1.0? And then check to see if it is behaving as you expect in master?

**Comment by arouinfar** [comment]
@jessegreenberg the behavior in master and rc.2 appears to be the same, and both are working as I would expect. The cloud reduces the surface temperature by 1 K and the reflected sunlight is thin but still noticeable.

**Comment by jessegreenberg** [comment]
OK thanks @arouinfar. If it is working correctly in both then perhaps https://github.com/phetsims/greenhouse-effect/commit/55963505b6a09fa0809ce1078495402bd8339d99 does not need to be in master. But Ill still leave up to @jbphet to confirm.  I will also check this issue off in https://github.com/phetsims/qa/issues/710

**Comment by jbphet** [comment]
Good catch @jessegreenberg - I had indeed missed moving one of the commits onto master.  I've done so now.

I tested this on master with the Gas Concentration slider at its nominal, min, and max positions and the cloud drops the temperature one degree Celsius and one degree Kelvin in each case.  I allowed the temperature to equilibrate for about 30 seconds prior to turning on the cloud in each case, and turned on the cloud for about 10 seconds, then turned it back off, and observed the temperature.

I think we're good here.  Closing.
